tests/test_feature.rb

```diff
@@ -97,7 +85,13 @@ def test_nv_overlay
       assert_raises(Cisco::UnsupportedError) { Feature.nv_overlay_enable }
       return
     end
+    # Test if VxLAN can be configured
+    vxlan_linecard?
+    vdc_current = node.product_id[/N7/] ? vdc_lc_state : nil
+    vdc_lc_state('f3') if vdc_current
     feature('nv_overlay')
+    # Return testbed to pre-clean state
+    vdc_lc_state(vdc_current) if vdc_current
```

Comment:
@chrisvanheuveln Aneesh's change is specific to N7k while mine fixes the test for N5k.  Since my PR will likely be merged last (still testing), I'll handle the merge conflict.